packages/app/app/components/InputDialog/index.tsx

```diff
@@ -2,7 +2,26 @@ import React, { useState, useCallback, useEffect } from 'react';
 import { Button, Input, Modal } from 'semantic-ui-react';
 import { useTranslation } from 'react-i18next';
 
-const InputDialog = ({ initialString, trigger, header, placeholder, accept, onAccept, testIdPrefix = null }) => {
+type InputDialogProps = {
+  
+initialString:string;
+trigger:JSX.Element;
```

Comment:
Could you use `ModalProps` from semantic ui, and `React.Element`? So these could be defined as

```typescript
trigger: ModalProps['trigger'];
header: React.Element;

```

Reply:
Made the Changes

Reply:
one more thing. How do I test locally because the log from `npm run build` isn't showing exactly what the problem might be ?
This is the log of the current build but doesn't show where the issue might have occurred even though the build failed. So is there a way I can test the locally without pushing commits ?
```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'build'
1 verbose cli ]
2 info using npm@6.14.11
3 info using node@v14.16.0
4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
5 info lifecycle @nuclear/app@0.6.17~prebuild: @nuclear/app@0.6.17
6 info lifecycle @nuclear/app@0.6.17~build: @nuclear/app@0.6.17
7 verbose lifecycle @nuclear/app@0.6.17~build: unsafe-perm in lifecycle true
8 verbose lifecycle @nuclear/app@0.6.17~build: PATH: C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\Coding Projects\nuclear\packages\app\node_modules\.bin;C:\Coding Projects\nuclear\packages\app\node_modules\.bin;C:\Coding Projects\nuclear\packages\node_modules\.bin;C:\Coding Projects\nuclear\node_modules\.bin;C:\Coding Projects\node_modules\.bin;C:\node_modules\.bin;C:\Program Files\nodejs;C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\Coding Projects\nuclear\node_modules\.bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\PuTTY\;C:\Program Files\MySQL\MySQL Shell 8.0\bin\;C:\Users\yahya.kayani\AppData\Local\Programs\Python\Python39\Scripts\;C:\Users\yahya.kayani\AppData\Local\Programs\Python\Python39\;C:\Users\yahya.kayani\AppData\Local\Microsoft\WindowsApps;C:\Users\yahya.kayani\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\yahya.kayani\AppData\Roaming\npm;C:\Users\yahya.kayani\AppData\Local\GitHubDesktop\bin;;C:\Program Files\JetBrains\PhpStorm 2020.1.4\bin;
9 verbose lifecycle @nuclear/app@0.6.17~build: CWD: C:\Coding Projects\nuclear\packages\app
10 silly lifecycle @nuclear/app@0.6.17~build: Args: [
10 silly lifecycle   '/d /s /c',
10 silly lifecycle   'cross-env NODE_OPTIONS=--max-old-space-size=8192 webpack --colors --progress --env.NODE_ENV=production'
10 silly lifecycle ]
11 silly lifecycle @nuclear/app@0.6.17~build: Returned: code: 2  signal: null
12 info lifecycle @nuclear/app@0.6.17~build: Failed to exec build script
13 verbose stack Error: @nuclear/app@0.6.17 build: `cross-env NODE_OPTIONS=--max-old-space-size=8192 webpack --colors --progress --env.NODE_ENV=production`
13 verbose stack Exit status 2
13 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:315:20)
13 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:315:20)
13 verbose stack     at maybeClose (internal/child_process.js:1048:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
14 verbose pkgid @nuclear/app@0.6.17
15 verbose cwd C:\Coding Projects\nuclear\packages\app
16 verbose Windows_NT 10.0.19042
17 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
18 verbose node v14.16.0
19 verbose npm  v6.14.11
20 error code ELIFECYCLE
21 error errno 2
22 error @nuclear/app@0.6.17 build: `cross-env NODE_OPTIONS=--max-old-space-size=8192 webpack --colors --progress --env.NODE_ENV=production`
22 error Exit status 2
23 error Failed at the @nuclear/app@0.6.17 build script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 2, true ]

```